### DISCUSSION TITLE
Flip video in webcam mode?

### DISCUSSION THREAD

**Discussion by ajbucci** [gtxaspec/wz_mini_hacks#213] [Q&A] [answered]
Does anyone know of a way to flip the video in webcam mode? Currently the video is up-side-down

**Comment by gtxaspec** [comment]
this is a bug in the ucamera application.  I will have to disassemble it in ghidra and see if this is possible.

**Comment by gtxaspec** [comment, marked as answer]
@ajbucci please try the latest master version, it should be fixed.  see #215

**Reply by ajbucci** [reply]
Fixed, thanks!!!
